### PR TITLE
Run record linkage migrations on spin up not RL endpoint request

### PR DESCRIPTION
**Pull request by gordonfarrell**
# PULL REQUEST

## Summary
Changing approach from last PR, not doing an endpoint, just running the migration function in main.py for record linkage. No need for a second PR in phdi-azure for this one.

## Related Issue
Fixes [#432](https://app.zenhub.com/workspaces/dibbs-63f7aa3e1ecdbb0011edb299/issues/gh/cdcgov/phdi/432)

## Additional Information
Anything else the review team should know?

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

**Review by nickclyde — APPROVED** [review]

File: containers/record-linkage/tests/test_record_linkage.py [hunk 1]
```diff
@@ -170,30 +179,4 @@ def test_linkage_success():
...
-
-        os.environ[setting] = "invalid_value"
-        get_settings.cache_clear()
-        actual_response = client.post("/link-record", json={"bundle": test_bundle})
-        assert actual_response.status_code == status.HTTP_400_BAD_REQUEST
-        assert "Could not connect to database" in actual_response.json()["message"]
-        os.environ[setting] = removed_setting
-
-    pop_mpi_env_vars()
+pop_mpi_env_vars()
```
Comment (DanPaseltiner) [inline containers/record-linkage/tests/test_record_linkage.py:182]: Is this actually doing anything for us given that all the other tests clean up the env after themselves? I'm probably missing something here.

**Review by DanPaseltiner — CHANGES_REQUESTED** [review]
This looks really good to me. Before we merge and automagically deliver a new version of this service straight to LAC, could we build the image and manually upload to Azure to confirm that these changes won't break anything? I these changes may cause a longer cold start and I'd like to make sure that the slower spin up speed doesn't cause any kind of inadvertent timeout issues in Azure.

File: containers/record-linkage/tests/test_record_linkage.py (hunk 1 above)
Comment (emmastephenson) [reply containers/record-linkage/tests/test_record_linkage.py:177]: +1, is this still needed?

**Review by emmastephenson — APPROVED** [review]
Looks good to me!

**Review by bamader — APPROVED** [review]
LGTM!

**Review by DanPaseltiner — APPROVED** [review]

**Pull request merged**